packages/medusa/src/services/event-bus.ts

```diff
@@ -346,9 +348,11 @@ export default class EventBusService {
         }
       })
 
-      await this.queue_.addBulk(eventsData).then(async () => {
-        return await stagedJobRepo.delete({ id: In(jobs.map((j) => j.id)) })
-      })
+      if (this.config_?.projectConfig?.redis_url) {
+        await this.queue_.addBulk(eventsData).then(async () => {
+          return await stagedJobRepo.delete({ id: In(jobs.map((j) => j.id)) })
+        })
+      }
```

Comment:
thought(non-blocking): Will this not bloat the staged job table? We are never deleting the jobs inserted in `emit`. Not super critical, as you would use Redis prod. environment, hence why its a non-blocking comment.

Reply:
I would argue that if the jobs are not processed they should not be removed. Wdyt? The user can still remohe them manually if necessary and as you said redis will be used in prod

Reply:
Yes, I agree. But imo. this is slightly different, as we deliberately ignore jobs now.

Reply:
But let's not do more for now. This is a temporary fix :)

Reply:
Yes agree, let’s wait the event module 👍